fecgov/openFEC
# update ofec_sched_e refresh script to specify data type for column sc…

…hedule_type

## Summary (required)

- Addresses # 3046

This problem is caused by one of the column in table ofec_sched_e has data type 'unknown', so when two cmte_ids are input and SQLAlchemy try to do the union, it throws an error and can not proceed.   The script that re-create the table ofec_sched_e had been updated to specify the column type.

## How to test the changes locally

- Script data/refresh/rebuild_schedule_e.sql can be run locally in the local database (the local database need to be up-to-date to have all the depending views).  After this sql script had been run, 
The API schedules/schedule_e should not throw error.

## Impacted areas of the application
List general components of the application that this PR will affect:

-  schedules/schedule_e

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
fix/other_pr | [link]()
feature/other_pr | [link]()
